index.html

```diff
@@ -0,0 +1,316 @@
+    </head>
+
+    <body>
+        <header>
+            <nav class="firstnav">
+                <i class="fa fa-bars fa-sm"></i>
+                <img src="https://www.freeiconspng.com/uploads/youtube-logo-png-transparent-image-5.png" alt="">
+            </nav>
+           
+            <form class="search">
+                <input type="text" placeholder="Search" value=""/>
+                <button><i class="fa fa-search"></i></button>
+            </form>
+
+            <i class="fa fa-upload fa-lg" aria-hidden="true"></i>
+            
+            <i class="fa fa-th"></i>
+
+            <i class="fa fa-user fa-lg"></i> 
+
+            <button class="sign-in">Sign in</button>
+
+        </header>
+        
+        <main>
+            <div class="msg">
+                <i class="fa fa-user fa-lg"></i>  
+                Choose your language. 
+                <i class="fa fa-times"></i>
+            </div>
+
+            <div class="lang-pref">  
+                You're viewing Youtube in <span><strong>English (US).</strong></span>    
+                You can <a href="">change this preference below.</a>  
+                <a href="" class="learn">Learn more</a>
+            </div>
+
+            <section class="vidcontent">
+                <div class="vid-des">
+                    <div class="videowrapper">
+                        <iframe width="560" height="315" src="https://www.youtube.com/embed/NOras8n86Fo" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>  
+                    </div>
+                    <div class="video-summary">
+                        <h2>Cristiano Ronaldo vs Lionel Messi 2020 ● Battle of the GOATS</h2>
+                        <div class="vs1">
+                            <div class="v">
+                                <div class="avatar">CR7</div>
+                                <div class="vs2">
+                                    <div class="title">Samuels Football</div> 
+                                    <div class="subscribe">
+                                        <button><i class="fa fa-youtube-play fa-lg" aria-hidden="true"></i>Subscribe</button>
+                                        <div class="num">799</div>
+                                    </div>
+                                </div>
+                            </div>
+                            
+                            <div class="views" style="display: flex; align-items: flex-end;">
+                                <span class="watch">50,016 views</span> 
+                            </div>
+                        </div>
+                        <hr>
+                        <div class="share">
+                            <a class="sh"><i class="fa fa-plus fa-lg" aria-hidden="true"></i>Add to</a>
+                            <a class="sh"><i class="fa fa-share fa-lg" aria-hidden="true"></i>Share</a>
```

Comment:
2. There should be a share button below the video content but currently, it is an icon.

Reply:
The share button below the video content is no more an icon. It is now a button.
Thanks. Awaiting approval.